src/Approximations/overapproximate.jl

```diff
@@ -111,6 +111,33 @@ function overapproximate(S::CartesianProductArray{N, <:AbstractHyperrectangle{N}
     return convert(Hyperrectangle, S)
 end
 
+"""
+    overapproximate(S::CartesianProduct{N, <:AbstractHyperrectangle{N}, <:AbstractHyperrectangle{N}},
+                    ::Type{<:Hyperrectangle}) where {N<:Real}
+
+Return a tight overapproximation of the cartesian product array of two
+hyperrectangles with one hyperrectangle.
```

Comment:
```suggestion
hyperrectangles by a new hyperrectangle.
```
Not sure ... I think this makes it a bit clearer that the product is for two boxes and we overapproximate with another box.